source/SIL.AppBuilder.Portal.Frontend/package.json

```diff
@@ -173,5 +173,9 @@
     "webpack-dev-server": "^3.1.4",
     "webpack-node-externals": "^1.7.2",
     "whatwg-fetch": "^3.0.0"
+  },
+  "resolutions": {
```

Comment:
This ensures we only have one copy of react + react-dom in our build.

Bundle Analysis:

![image](https://user-images.githubusercontent.com/199018/49667982-c8d87700-fa2a-11e8-907b-778163aabde1.png)

auth0-lock is currently: 103KB gzipped, and before it was 173KB gzipped